ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -519,7 +527,40 @@ try:
 
             security_group_ids = args.securityGroupIds
             if not security_group_ids:
-                security_group_ids = setup.create_security_group()
+                if not args.accountName:
+                    security_group_ids = setup.create_security_group()
+                else:
+                    try:
+                        security_group_ids = utils.get_from_config(
+                            args.accountName,
+                            config,
+                            region,
+                            'security_group_ids_%s' % region,
+                            '--security-group-ids'
+                        )
+                        if args.verbose and security_group_ids:
+                            msg = 'Using Seccurity Group IDs: %s'
+                            print(msg % security_group_ids)
+                    except Exception:
+                        if args.verbose:
+                            msg = 'No security group specified in the '
+                            msg += 'configuration, "security_group_ids_%s" or '
+                            msg += 'given on the command line.'
+                            print(msg % region)
+                    if not security_group_ids and vpc_subnet_id:
+                        ec2 = boto3.client(
+                            aws_access_key_id=access_key,
+                            aws_secret_access_key=secret_key,
+                            region_name=region,
+                            service_name='ec2'
+                        )
+                        subnet_data = ec2.describe_subnets(
+                            SubnetIds=[vpc_subnet_id]
+                        )
+                        vpc_id = subnet_data['Subnets'][0]['VpcId']
```

Comment:
This assumes the subnet exists. Maybe good to catch IndexError and handle accordingly.

Reply:
I'm good either way; if it doesn't exist, the error _should_ bubble up.